isort/settings.py

```diff
@@ -345,7 +345,7 @@ def __init__(
                     "was found inside. This can happen when [settings] is used as the config "
                     "header instead of [isort]. "
                     "See: https://pycqa.github.io/isort/docs/configuration/config_files"
-                    "/#custom_config_files for more information."
+                    "#custom_config_files for more information."
```

Comment:
```suggestion
                    "#custom-config-files for more information."
```

This is the new pattern.